src/components/Common/Button/index.tsx

```diff
@@ -74,6 +75,9 @@ function Button<P extends ElementTypes = 'button'>(
         'text-white bg-transparent border-gray-600 hover:border-gray-200 focus:border-gray-100 active:border-gray-100'
       );
       break;
+    case 'plex':
+      buttonStyle.push('plex-button');
+      break;
```

Comment:
Seems like we aren't using this anymore?

Reply:
It's used by the plex PlexLoginButton component.

Reply:
I don't see it being used?

Reply:
https://github.com/sct/overseerr/blob/3dc7b8e04e5fb0c00697165960beec5f24d0c4f8/src/components/PlexLoginButton/index.tsx#L32